ui/components/multichain/asset-picker-amount/asset-picker-amount.tsx

```diff
@@ -62,14 +65,27 @@ export const AssetPickerAmount = ({
   const isSwapsErrorShown = isDisabled && swapQuotesError;
 
   const [isFocused, setIsFocused] = useState(false);
+  const [isNFTInputChanged, setIsTokenInputChanged] = useState(false);
+
+  const handleChange = useCallback(
+    (newAmountRaw, newAmountFormatted) => {
+      if (!isNFTInputChanged && asset.type === AssetType.NFT) {
+        setIsTokenInputChanged(true);
+      }
```

Comment:
Do you think this state should be reset when the selected NFT token changes?

Reply:
Yeah good call – updated